src/app/components/Hand/Hand.tsx

```diff
@@ -1,19 +1,148 @@
+import { useRef, useState } from 'react'
 import Box, { BoxProps } from '@mui/material/Box'
+import useTheme from '@mui/material/styles/useTheme'
 
+import { Card } from '../Card'
+import { mathService } from '../../../services/Math'
 import { lookup } from '../../../game/services/Lookup'
+import { UnimplementedError } from '../../../game/services/Rules/errors'
+import * as cards from '../../../game/cards'
 import { IGame, IPlayer } from '../../../game/types'
+import { isCardId } from '../../../game/types/guards'
+import { CARD_HEIGHT } from '../../config/dimensions'
+
+export const selectedCardTransform = `translateX(-50%) translateY(0) rotate(0deg) scale(1) rotateY(0deg)`
+
+const deselectedIdx = -1
+const foregroundCardScale = 1
+const backgroundCardScale = 0.65
+const foregroundCardZIndex = 20
+
+export const getGapPixelWidth = (numberOfCards: number) => {
```

Comment:
it looks to me like you could use some simple math here and basically end up with the exact same results.

if you consider 50 the base, and divide by the number of cards, then multiple by 3 you get more or less the same values with less maintenance. 

i just did this in a node console to check
```
const BASE_GAP = 50
const samples = [60, 30, 20, 10, 5, 3]
const getGap = (numCards) => Math.min(BASE_GAP, Math.ceil(BASE_GAP / numCards
* 3))
samples.map(getGap)
// outputs: [ 3, 5, 8, 15, 30, 50 ]
```

Reply:
I appreciate the conciseness of this alternative approach, but I'd like to stick with the solution that's already in place. Such a primitive approach is (subjectively, to me) easier to understand, but more importantly it doesn't tie this behavior to a mathematical algorithm.

To put it another way, each step is manually defined as something that _feels_ aesthetically good (to me). This function deals with a display detail where the result "feeling right" is more important than mathematical correctness[^1].

[^1]: I think there's a term for this that's escaping me at the moment. [Here's a ChatGPT conversation about Intuitive Design that articulates what I'm getting at.](https://chat.openai.com/share/6165e40a-6089-40df-a437-772f02b79bc0)